### DISCUSSION TITLE
Why this issue was marked as resolved?

### DISCUSSION THREAD

**Discussion by omariom** [dotnet/wpf#7163] [General]
https://github.com/dotnet/wpf/issues/4181

**Comment by lindexi** [comment]
@omariom See https://github.com/dotnet/wpf/issues/4181#issuecomment-781706675

Feel free to re-create the new issues.

**Comment by omariom** [comment]
@lindexi 
Why should I?
That issue perfectly describes the problem.
It should be just reopened.

**Comment by miloush** [comment]
I am not really keen on accommodating 3rd party software that messes things up. Where would that end?
